person.rb

```diff
@@ -1,16 +1,17 @@
-require './nameable'
+require './decorators/nameable'
 require 'securerandom'
 
 class Person < Nameable
-  attr_reader :id
-  attr_accessor :name, :age
```

Comment:
- [ ] The `Book` and the `Person` class have a many-to-many relationship through the `Rental` class. Therefore, each class instance needs to keep track of its rentals. Kindly implement this in this class. 👍 💻 

#### Hint:

```ruby
def add_rental(book, date)
  Rental.new(date, book, self)
end
```